roundabout-core/src/test/java/at/fhv/itm3/s2/roundabout/entity/StreetSectionTest.java

```diff
@@ -58,7 +83,34 @@ public void firstCarCouldEnterNextSection() throws Exception {
     }
 
     @Test
-    public void isEnoughSpace() throws Exception {
+    public void firstCarCouldEnterNextSection_everythingOK() throws Exception {
```

Comment:
everything ok does not say much, i would use a better name, e.g. _carCanEnterNextSection